backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/utils/JavaScriptBindings.java

```diff
@@ -0,0 +1,22 @@
+package org.molgenis.emx2.utils;
+
+import java.util.Map;
+import org.molgenis.emx2.graphql.GraphqlApiFactory;
+
+public class JavaScriptBindings {
+
+  @FunctionalInterface
+  public interface SimplePostClient {
+    Object execute(String query, Map<String, Object> variables, String schemaId);
+  }
+
+  public static SimplePostClient simplePostClient =
```

Comment:
You will need a graphql instance that has the current database instance. That might make it bit more difficult.
I.e. you probably would need to instantiate this simplePostClient within the insert (ideally once for the transaction)